bokeh/models/mappers.py

```diff
@@ -16,6 +16,17 @@ class ColorMapper(Model):
 
     """
 
+    palette = Seq(Color, help="""
+    A sequence of colors to use as the target palette for mapping.
+
+    This property can also be set as a ``String``, to the name of
+    any of the palettes shown in :ref:`bokeh.palettes`.
+    """).accepts(Enum(Palette), lambda pal: getattr(palettes, pal))
+
+    def __init__(self, palette=None, **kwargs):
+        if palette is not None: kwargs['palette'] = palette
+        super(ColorMapper, self).__init__(**kwargs)
+
```

Comment:
Is there a possibility of a `ColorMapper` that does not have a "list of colors" type `palette`? I can sort of imagine that there could possibly be a more general class of color mappers, but I guess we can cross that bridge if it ever happens.